### ISSUE TITLE
Build fails on Xcode 8.0

### ISSUE THREAD

**Issue by roberttpowell** [frenetisch-applaudierend/ios-drag-and-drop#10]
Using Cocoapods for installation, the build fails in DNDDragAndDropController_private.h

with DNDDragAndDropController.h not found.

Removing the following lines from the module map enables the build to run

private header "DNDDragAndDropController_Private.h"
private header "DNDDragOperation_Private.h"
private header "DNDDragHandler.h"

but this is probably not the best solution.

**Comment by frenetisch-applaudierend** [comment]
Hi Robert

Thanks for the report. Just so I can try to reproduce the problem:

* Do I correctly assume the project is in Swift? Which version do you use?
* What version of cocoapods do you use?

**Comment by roberttpowell** [comment]
Hi Markus,

Thanks for the quick response. Yes it is a mixed Objective C and Swift 3.0 project.

I am using cocoapods as follows

0.39.0
cocoapods-plugins: 0.4.2
cocoapods-search: 0.1.0
cocoapods-stats: 0.6.2
cocoapods-try: 0.5.1
cocoapods-trunk: 0.6.4
cocoapods-plugins: 0.4.2
cocoapods-trunk: 0.1.4
cocoapods-try: 0.5.1

**Comment by frenetisch-applaudierend** [comment]
Sorry for the long delay (holidays 😎)

I tried now to reproduce the issue, so far no luck. I created an empty sample project (Xcode 8), added an objc class and a bridging header and added the pod. This seems to work.

Here is the Podfile I used:
```
use_frameworks!

target 'SampleProject' do
  pod 'DNDDragAndDrop', '~> 1.2'
end
```

This is the generated module map:

```
framework module DNDDragAndDrop {
  umbrella header "DNDDragAndDrop-umbrella.h"

  export *
  module * { export * }
}
```

I did however use a newer version of cocoapods (1.0.1), so maybe this is a bug in cocoapods that was fixed. I'll try to get a setup with the version you mentioned and see if I can reproduce it.

Any chance you can switch to cocoapods 1.0 to see wether this solves your problem?

**Comment by roberttpowell** [comment]
I was stuck on the older version of cocoa pods prior to getting a release out. It has now gone, and cocoa pods upgrade is on our todo list for the next bus stop, so I will give that a try. Prior to upgrading I will see if I can create a simple project that exhibits the problem that I can share.

**Comment by roberttpowell** [comment]
I have an isolation case showing the problem built using the cocoa pods versions above. The attached zip file contains an almost empty Xcode 8 project that fails to build with this problem.

[TestDND.zip](https://github.com/frenetisch-applaudierend/ios-drag-and-drop/files/511510/TestDND.zip)

**Comment by roberttpowell** [comment]
I have upgraded to cocoapods 1.0.1 and as a result get the same module map as you. My Swift bridging header has the following line 

`#import <DNDDragAndDrop/DNDDragAndDrop.h>` 

which gives me a file not found error.

All of this can be seen in the attached Xcode workspace
[TestDND 2.zip](https://github.com/frenetisch-applaudierend/ios-drag-and-drop/files/511556/TestDND.2.zip)

**Comment by frenetisch-applaudierend** [comment]
Thanks for the samples.

Interestingly I see different results. Your second sample project works fine for me (compiles and runs). The first one fails building with the error you mentioned. Maybe a mixup (I noticed the folders have the same name, that got me the first time :D)?

If my observations match yours, I would be tempted to conclude that this is in fact a cocoapods bug that got fixed. Not sure I could do something to fix this.
